config/confignet/confignet.go

```diff
@@ -58,6 +68,11 @@ type TCPAddrConfig struct {
 	DialerConfig DialerConfig `mapstructure:"dialer"`
 }
 
+// set default values for the TCP Address configuration
+func NewDefaultTCPAddrConfig() TCPAddrConfig {
+	return TCPAddrConfig{}
```

Comment:
Any reason to use the `NewDefaultDialerConfig` in `NewDefaultAddrConfig` but not in `NewDefaultTCPAddrConfig`?

Reply:
Yes, sorry I overlooked that struct's  structure. I have corrected the same.